docker-compose.yml

```diff
@@ -0,0 +1,43 @@
+version: '3'
+
+services:
+  web:
+    image: nginx
+    volumes:
+      - ./templates:/etc/nginx/templates
+    ports:
+      - "8000:80"
+    environment:
+      - NGINX_HOST=gamemuster.com
+      - NGINX_PORT=80
+
+  db:
+    image: postgres:alpine
+    environment:
+      POSTGRES_DB: gamestore
+      POSTGRES_USER: gamestoreuser
+      POSTGRES_PASSWORD: gamestorepassword
+    restart: always
+    expose:
+      - "5432"
+
+  rabbitmq:
+    image: rabbitmq:alpine
+
+  app:
+    build:
+      context: .
+      dockerfile: Dockerfile-gamestore
+    environment:
+      - DB_NAME=gamestore
+      - DB_USER=gamestore
+      - DB_PASSWORD=gamestore
+      - DB_PORT=5432
+    links:
+      - db:db
+    ports:
+      - "8000:8000"
+    depends_on:
+      - db
+      - rabbitmq
+      - web
```

Comment:
Actually you should start nginx service with proxy pass to app service, so nginx should depend on app.